collector/lib/NetworkStatusNotifier.cpp

```diff
@@ -305,7 +305,16 @@ void NetworkStatusNotifier::AddConnections(::google::protobuf::RepeatedPtrField<
   int64_t per_container_limit = config_.PerContainerRateLimit();
   CountLimiter rate_limiter(per_container_limit);
 
+  CLOG(DEBUG) << "Per container rate limit: " << per_container_limit;
+
+  UnorderedMap<std::string, int> rate_limited_containers;
```

Comment:
Since the keys used in this container are coming from the `delta` argument and they both live for the duration of this call, could we use `const std::string&` as the key and avoid additional copies?

Reply:
Done